_posts/2021-03-16-notes-from-community-call-4-march-2021.md

```diff
@@ -0,0 +1,72 @@
+## Proposed for this call
+
+We've had requests to review codebases against the Standard from codebases that at the time were not on a path towards full stewardship. We need to prioritize our work, but we also want the Standard for Public Code to be widely used.
+
+We've already created [an introduction course](https://citybycity.academy/course/standards-for-smart-cities) and briefly floated an idea of vendor training.
+
+Let's have an open discussion now on how to grow a community that's confident in applying the Standard for Public Code to a codebase.
+
+## Notes
+
+The discussion had a lot of different angles and ideas. Below are a few of those that sparked most discussion.
+
+### The standard as a help in procurement
+
+We discussed if the standard could be a help for a public organization about to procure development services. Right now the confidence of both buyer and the seller are too low for either of them to go that way. In theory, the standard should be able to be part of a procurement, much like a set of functional requirements. What might help are more codebases showing how they are compliant, and guides to get there.
+
+### "Flavors" of the standard
+
+Another idea that was discussed was "flavors" of the standard that certain solutions and practices are perscribed in a stronger sense than the standard is today. The openess of it now gives so many choices for the implementing party it doesn't really give much direction at all. If there were some framework specific flavors that perscribed certain styles, tools and methods, it would be easier to start developing and spend less time choosing amongst these.
+
+### Generic codebase template
+
+Related to this, but also different in the approach, one could imagine a very generic codebase (which in itself doesn't do much) that meets the standard that could be used as a reference point and inspiration. This could have the most preferred of all methods and good practices in it. But it might not be the Foundation for Public Code's role to maintain it or decide what goes in it, but rather it could be a community curated resource.
+
+### Guides
+
+More guides like the one on [how to start a Standard compliant GitHub repository](https://blog.publiccode.net/news/2020/12/09/how-to-start-a-standard-compliant-repository.html) was also requested.
+
+### Vendors checking compliance
+
+Could checking standard compliance be done by vendors? Yes, that should be possible. However, only the Foundation for Public Code can certify a codebase, but as a pre-check that should be doable. Vendors might be interested in selling this service if there is large enough demand for it.
+
+### Marketing and outreach
+
+To get more people ver the first threshold, and to establish the Standard for Pubic Code as a well known concept, there were suggestions to make more marketing of it. Unfortunately we didn't have time to discuss how to do it at any length, sout it might be topic for a separate community call in the future.
```

Comment:
```suggestion
To get more people over the first threshold, and to establish the Standard for Public Code as a well known concept, there were suggestions to market it more proactively. Unfortunately we didn't have time to discuss how to do this in detail, though it might be topic for a separate community call in the future.
```